salt/beacons/__init__.py

```diff
@@ -129,6 +129,25 @@ def add_beacon(self, name, beacon_data):
 
         return True
 
+    def modify_beacon(self, name, beacon_data):
+        '''
+        Modify a beacon item
```

Comment:
I can add the versionadded tag but I'm not sure it makes sense to add the docs for the parameters here, they're internal functions that should only be called via the event system.